### ISSUE TITLE
Add "Save as" button

### ISSUE THREAD

**Issue by lana-k** [lana-k/sqliteviz#16]

**Comment by ajete** [comment]
I came across this tool and I used it but without exporting the option to csv query results I can't fully use it on a daily basis, do you know how difficult is to add if this feature?
Thanks

**Comment by lana-k** [comment]
Yes, I have this feature (exporting a result set into CSV) planned for one of the next versions.
Also, it would be useful to know how you are going to use this feature (and sqliteviz in general on a daily basis :-)) in order to make the right assumptions during its implementation.

**Comment by twoxfh** [comment]
@lana-k csv export would be great, individual column export would help too for json documents. Not sure how to implement with all the great pivot table changes

**Comment by lana-k** [comment]
@ajete Export result set to CSV has just released (v0.16.0).
